### ISSUE TITLE
6.0.6 broke support for yarn workspaces

### ISSUE THREAD

**Issue by Yzrsah** [ds300/patch-package#133]
patch-package previously worked with yarn workspaces.

It stopped working in 6.0.6.

```
patch-package 6.0.6

**ERROR** No package-lock.json, npm-shrinkwrap.json, or yarn.lock file.
```

The reason is because these lock files do not exist in the package directories.

patch-package 6.0.5 is fully working with yarn workspaces, working in all package directories, working with `yarn` or `yarn install` from any package directory (not just root) with all dependencies and install steps updated correctly and without issues for commands at any package level.

**Comment by ds300** [comment]
Hi 👋 Thanks for the report! 

I think I know why this happened. Will publish a fix asap.

**Comment by ds300** [comment]
Just pushed out a fix in 6.0.7. Let me know if that works for you!

**Comment by Yzrsah** [comment]
Thank you very much for the update, it is fixed.
